Automattic/vip-go-mu-plugins
# Hide the Instant Search toggle in Jetpack search settings

## Description
Part 2 of the changes in #1548 to hide settings related to Instant search.  With the addition of [CSS IDs on the search cards](https://github.com/Automattic/jetpack/pull/15440), we can now more easily target individual settings. This PR hides the Instant search toggle using the `jp-settings-search-jetpack` ID  

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Pull down PR locally or to sandbox
1. Navigate to Jetpack Search Setting screen (`/wp-admin/admin.php?page=jetpack#/performance`)
1. Under the Search section under the "Enable Search" toggle, confirm that this toggle no longer appears: "Enable instant search experience (recommended)"
![jp-search2](https://user-images.githubusercontent.com/822209/81237023-5e2ac280-8fb3-11ea-9d2b-7e9e05eb4eab.png)
